spackbot/handlers/labels.py

```diff
@@ -68,7 +68,7 @@
     "external-packages": {"patch": r"[+-] +def determine_spec_details\("},
     "libraries": {"patch": r"[+-] +def libs\("},
     "headers": {"patch": r"[+-] +def headers\("},
-    "stand-alone-tests": {"patch": r"[+-] +def test\("},
+    "stand-alone-tests": {"patch": r"[+-] +def test[_]?.*\("},
```

Comment:
I think this one marks any PR adding a unit-test as a `stand-alone-tests`, see e.g. https://github.com/spack/spack/pull/43143

Reply:
Side-effect of the name change to conform to the pytest naming convention.  Should only be applying in this case if NOT in the `test` subdirectory of core.